spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/data/vectorized/IcebergArrowColumnVector.java

```diff
@@ -59,6 +59,12 @@ public void close() {
     accessor.close();
   }
 
+  @Override
```

Comment:
Did we land on adding the `@Override`? Now it doesn't compile against Spark 3.5.2. Leaving out the annotation is safer (it will still work with ≥3.5.4).

Reply:
> Did we land on adding the `@Override`? Now it doesn't compile against Spark 3.5.2. Leaving out the annotation is safer (it will still work with ≥3.5.4).

+1

Reply:
removed.

both keeping and removing are fine to me, it has no difference in runtime